modules/microshift-exposed-audit-ports-hostnetwork.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-exposed-audit-ports-hostnetwork_{context}"]
+
+== hostnetwork
+
+When a pod is configured with the `hostNetwork:true` setting, the pod is running in the host network namespace. This configuration can independently open host ports. {microshift-short} component logs cannot be used to track this case, the ports are subject to firewalld rules. If the port opens in firewalld, you can view the port opening in the firewalld debug log.
+
+.Prerequisites 
+
+* You have root user access to your build host.
+
+.Procedure
+
+. Optional: You can check that the `hostnetwork:true` parameter is set in your ovnkube-node pod by running the following command
++
+[source,terminal]
+----
+$ sudo oc get pod -n openshift-ovn-kubernetes ovnkube-node-7mmlp -o json | jq -r '.spec.hostNetwork' true
+----
+
+. Enable debug in the firewalld log by running the following command:
++
+[source,terminal]
+----
+$ sudo vi /etc/sysconfig/firewalld
+FIREWALLD_ARGS=--debug=10
+----
+
+. Restart the firewalld service
++
+[source,terminal]
+----
+$ sudo systemctl restart firewalld.service
```

Comment:
The user can verify the debug variable was added by running 
`sudo systemd-cgls -u firewalld.service`

`[redhat@dhcp-1-235-9 ~]$ sudo systemd-cgls -u firewalld.service
Unit firewalld.service (/system.slice/firewalld.service):
└─21776 /usr/bin/python3 -s /usr/sbin/firewalld --nofork --nopid --debug=10`